src/utils/oppfolgingstilfelleUtils.ts

```diff
@@ -0,0 +1,56 @@
+import { OppfolgingstilfelleDTO } from "@/data/oppfolgingstilfelle/person/types/OppfolgingstilfellePersonDTO";
+import dayjs from "dayjs";
+import {
+  MIN_DAYS_IN_LONG_TILFELLE,
+  THREE_YEARS_AGO_IN_MONTHS,
+} from "@/data/oppfolgingstilfelle/person/oppfolgingstilfellePersonQueryHooks";
+import { dagerMellomDatoer } from "@/utils/datoUtils";
+
+const daysInTilfelle = (tilfelle: OppfolgingstilfelleDTO) => {
+  return dagerMellomDatoer(tilfelle.start, tilfelle.end) + 1;
+};
+
+const hasManySykefravar = (tilfeller: number, sickdays: number) => {
+  return tilfeller > 4 && sickdays > 100;
+};
+
+const hasLongSykefravar = (tilfeller: number, sickdays: number) => {
+  return tilfeller > 1 && sickdays > 300;
+};
```

Comment:
Dette var de tingene som Stine hadde jobbet med å definere? 🤗

Reply:
Det stemmer! I tillegg til reglene om at vi ikke bryr oss om tilfeller som sluttet for mer enn tre år siden, eller er mindre enn 4 dager lange.